cmd/gcp/main.go

```diff
@@ -76,16 +76,38 @@ func main() {
 	flag.Parse()
 	ctx := context.Background()
 
+	timeSource := sctfe.SystemTimeSource{}
 	signer, err := NewSecretManagerSigner(ctx, *signerPublicKeySecretName, *signerPrivateKeySecretName)
 	if err != nil {
 		klog.Exitf("Can't create secret manager signer: %v", err)
 	}
+	cpSigner, err := sctfe.NewCpSigner(signer, *origin, timeSource)
+	if err != nil {
+		klog.Exitf("failed to create checkpoint Signer: %v", err)
+	}
+
+	storage, err := newGCPStorage(ctx, cpSigner)
+	if err != nil {
+		klog.Exitf("failed to initiate storage backend: %v", err)
+	}
 
 	vCfg, err := sctfe.ValidateLogConfig(*origin, *rootsPemFile, *rejectExpired, *rejectUnexpired, *extKeyUsages, *rejectExtensions, notAfterStart.t, notAfterLimit.t, signer)
 	if err != nil {
 		klog.Exitf("Invalid config: %v", err)
 	}
 
+	opts := sctfe.HandlerOptions{
+		Validated:          vCfg,
+		Deadline:           *httpDeadline,
+		MetricFactory:      prometheus.MetricFactory{},
+		RequestLog:         new(sctfe.DefaultRequestLog),
```

Comment:
nit: match style for `MetricFactory`: 

```suggestion
		RequestLog:         &sctfe.DefaultRequestLog{},
```